hw/top_earlgrey/ip_autogen/rv_plic/data/rv_plic.hjson

```diff
@@ -1,11 +1,3 @@
-// Copyright lowRISC contributors.
-// Licensed under the Apache License, Version 2.0, see LICENSE for details.
-// SPDX-License-Identifier: Apache-2.0
-//
-// ------------------- W A R N I N G: A U T O - G E N E R A T E D   C O D E !! -------------------//
```

Comment:
could this potentially use the autogen header library that @sriyerg has added recently?

Reply:
None of the files in the ip_autogen folder have autogen headers:

* All files in ip_autogen are auto-generated, I don't want to give the impression that only some of them are.
* Adding headers to all files is tricky: ipgen doesn't know and care about file types; adding generated headers would require per-extension logic to e.g. use the right types of comments.
* We were talking about symlinking some files in ip_autogen to ip_templates, which would prevent added headers.

I'd therefore much prefer developers getting used to the fact that all in ip_autogen (as in other autogen folders) is autogenerated and shouldn't be manually edited. (And ultimately, maybe we can get rid of the autogenerated checked-in files completely, but that's a separate discussion.)